core/graph/src/main/java/uk/gov/gchq/gaffer/graph/GraphConfig.java

```diff
@@ -85,6 +86,14 @@ public void setLibrary(final GraphLibrary library) {
         this.library = library;
     }
 
+    public String getDescription() {
+        return description;
+    }
+
+    public void setDescription(final String description) {
+        this.description = description;
```

Comment:
I think we will leave any restrictions on the description currently as it isn't 'used' anywhere.  We can look into updating this in the future though if we need to.